shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/dml/select-special-function.xml

```diff
@@ -211,4 +211,92 @@
             <simple-table name="t_order" start-index="29" stop-index="35" />
         </from>
     </select>
+    
+    <select sql-case-id="select_ifnull">
```

Comment:
This PR should not change the SQL parsing, so there is no need to add additional SQL Parser test cases.